projects/Mallard/src/helpers/download-todays-issue.ts

```diff
@@ -0,0 +1,15 @@
+import { prepFileSystem, clearOldIssues, downloadTodaysIssue } from './files'
+import { fetchCacheClear } from './fetch'
+
+const clearAndDownloadIssue = async () => {
+    await prepFileSystem()
+    await clearOldIssues()
+    const weOk = await fetchCacheClear()
+    if (weOk) {
+        return await downloadTodaysIssue()
+    } else {
+        return Promise.resolve()
```

Comment:
just remove the whole `else` clause? which would be equivalent afaik

Reply:
fixed